src/pages/use-less-energy/index.js

```diff
@@ -406,7 +409,7 @@ const UseLessEnergyPage = () => (
               <li>Use less hot water and turn stuff off</li>
               <li>Vote for those who prioritize the planet</li>
               <li>
-                <a href="#share" alt="Spread the word">
+                <a href="use-less-energy/#share" alt="Spread the word">
```

Comment:
can you remove the `use-less-energy/` - it will make a smooth scroll to the section this way :)